File: level35/homework/hw1.py
#1) რა არის ფუნქცია?
#ფუნქცია არის პატარა „მანქანა“, რომელსაც აძლევ რაღაცას და ის გიბრუნებს პასუხს.
#მაგალითად: აძლევ რიცხვს → გიბრუნებს მის გაორმაგებულს.

#2
def check(num):
 if num % 2 == 0:
   print("Even")
 else:
   print("Odd")
check(5)
check(10)


#3 
def sum_positive(numbers):
 total = 0
 for n in numbers:
        if n > 0:
            total += n
 return total
print(sum_positive([2,1,3,5,6]))

File: level58/homework/hw1.js
// 1)
function greet(name) {
  console.log(`Hello, ${name}`);
}

greet("Giorgi");


// 2)
function umciresinum(arr) {
  let min = arr[0];

  for (let i = 1; i < arr.length; i++) {
    if (arr[i] < min) {
      min = arr[i];
    }
  }

  console.log(min);
}

umciresinum([2, 10, -20, 100, 90, 50, 40]);


// 3)
function numbers(num) {
  if (num === 0) {
    console.log("Zero");
  } else if (num % 2 === 0) {
    console.log(num ** 3);
  } else {
    console.log(num ** 2);
  }
}

numbers(5);
numbers(4);
numbers(0);


// 4)
function pasi(price, discount) {
  let result = price - (price * discount) / 100;

  console.log(result);
}

pasi(100, 20);


// 5)
function sumNumbers(arr) {
  let evenSum = 0;
  let oddSum = 0;

  for (let i = 0; i < arr.length; i++) {
    if (arr[i] % 2 === 0) {
      evenSum += arr[i] ** 2;
    } else {
      oddSum += arr[i] ** 3;
    }
  }

  console.log("Even Sum:", evenSum);
  console.log("Odd Sum:", oddSum);
}

sumNumbers([1,2,3,4,5,6,7,8,9,10]);